ArduPlane/quadplane.cpp

```diff
@@ -1292,3 +1300,37 @@ bool QuadPlane::verify_vtol_land(const AP_Mission::Mission_Command &cmd)
     check_land_complete();
     return false;
 }
+
+void QuadPlane::enable_geofence(void)
```

Comment:
I just updated this, would you be able to take a look? I added a bool return but did not use the bool in the calling functions as there is not much handling to be done on a failed geofence enable/disable. When it happens it is usually caused by misconfigured geofence.